CHANGELOG/changelog

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env lua
+
+local pl_template = require "pl.template"
+local pl_tablex = require "pl.tablex"
+local pl_file = require "pl.file"
+local pl_dir = require "pl.dir"
+local pl_path = require "pl.path"
+local pl_stringx = require "pl.stringx"
+local lyaml = require "lyaml"
+local pl_app = require 'pl.lapp'
+
+local PATH_UNRELEASED = "unreleased"
+local GITHUB_REPO = "Kong/kong"
+local CHANGELOG_ENTRY_TEMPLATE = "changelog-template.yaml"
+local JIRA_BASE_URL = "https://konghq.atlassian.net/browse/"
```

Comment:
Regarding the link to JIRA issue: is this intended for internal audience? We don't reference the JIRA tickets in the docs site changelog.

Reply:
The CHANGELOG-EE.md include the JIRA ticket in every item, so I was trying to keep the new release note as original as possible. It sounds like you want to use the generated release note to be directly used in the public docs site, I'm happy to try and help. But I think the docs team still needs to review and even edit the generated release note before publishing it to the docs site.


So, in short, questions:
1. Do you think the generated release note should be directly used in the docs site without edit/optimize?
(Currently, the generated released note is more for internal audience)